pasqal_cloud/__init__.py

```diff
@@ -214,6 +216,58 @@ def cancel_job(self, id: str) -> Job:
         job = Job(**job_rsp, _client=self._client)
         return job
 
+    def create_workload(
+        self,
+        workload_type: str,
+        backend: str,
+        config: Dict[str, Any],
+        wait: bool = False,
+    ) -> Workload:
+        req = {
+            "workload_type": workload_type,
+            "backend": backend,
+            "config": config,
+        }
+        workload_rsp = self._client._send_workload(req)
+        workload_id = workload_rsp["id"]
+        if wait:
+            while workload_rsp["status"] in ["PENDING", "RUNNING"]:
+                time.sleep(RESULT_POLLING_INTERVAL)
+                workload_rsp = self._client._get_workload(workload_id)
```

Comment:
nit: we could refactor to avoid having this code twice.


```
def _wait_for_workload(workload_resp: dict[str, Any], workload_id: str):
    while workload_rsp["status"] in ["PENDING", "RUNNING"]:
                 time.sleep(RESULT_POLLING_INTERVAL)
                 workload_rsp = self._client._get_job(id)
    return workload_rsp
```